auslib/web/admin/views/required_signoffs.py

```diff
@@ -48,6 +50,19 @@ def __init__(self, table, decisionFields):
         self.decisionFields = decisionFields
         super(RequiredSignoffsHistoryAPIView, self).__init__(table=table)
 
+    def _get_filters(self):
+        query = get_input_dict()
+        where = [getattr(self.table.history, f) == query.get(f) for f in query]
+        where.append(self.table.history.data_version != null())
+        if hasattr(self.history_table, 'product'):
+            where.append(self.history_table.product != null())
```

Comment:
I'm not sure I understand this block. All required signoffs tables have a required column of product, so it will always be in every row of the history table. Are you sure this is useful?

Reply:
I think it's redundant. I'll remove it. Thank you!